tms_expense/models/res_partner.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    employee_id = fields.Many2one("hr.employee")
```

Comment:
Here, I also believe that there is no need to create a new field. You can probably retrieve this based on the existing data.

Reply:
Maybe search for the partner in employee_id.address_home_id (where address_home is a res.partner) or, if it is a user, get the employee_id and partner_id linked to the user.